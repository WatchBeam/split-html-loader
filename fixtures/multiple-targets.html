<!-- platform: xbox -->
<a></a>
<!-- locale: not-en -->
<b></b>
<!-- locale: en -->
<c></c>
<!-- platform: not-desktop -->
<d></d>
