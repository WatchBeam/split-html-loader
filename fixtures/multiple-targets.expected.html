<!-- platform: xbox -->
<a></a>
<!-- locale: not-en --><!-- 2 nodes snipped by split-html -->
<!-- locale: en -->
<c></c>
<!-- platform: not-desktop -->
<d></d>
